client/src/components/RequestForm/RequestForm.tsx

```diff
@@ -0,0 +1,156 @@
+import { Autocomplete, Box, Card, Grid, InputLabel, TextField, Typography } from '@mui/material';
+import { Profile } from '../../interface/Profile';
+import { Star } from '@mui/icons-material';
+import { useStyles } from './useStyles';
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Form, Formik, FormikHelpers } from 'formik';
+import { DatePicker, LocalizationProvider } from '@mui/lab';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+
+interface Props {
+  className?: string;
+  profile: Profile;
+  handleSubmit: (
+    {
+      dropInDate,
+      dropInTime,
+      dropOffDate,
+      dropOffTime,
+    }: {
+      dropInDate: Date;
+      dropInTime: string;
+      dropOffDate: Date;
+      dropOffTime: string;
+    },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      dropInDate: Date;
+      dropInTime: string;
+      dropOffDate: Date;
+      dropOffTime: string;
+    }>,
+  ) => void;
+}
+
+const getTimes = (hour: number) => {
+  const times = [];
+  for (let i = hour; i < 24; i++) {
+    times.push(`${i % 12 ? i % 12 : 12} ${i / 12 < 1 ? 'am' : 'pm'}`);
+  }
+  return times;
+};
+
+const RequestForm = ({ className, profile, handleSubmit }: Props): JSX.Element => {
+  const classes = useStyles();
+  const today = new Date();
+
+  return (
+    <Card elevation={8} className={className}>
+      <Grid container direction="column" alignItems="center" marginTop={3}>
+        <Typography variant="h5" sx={{ fontWeight: 'bold' }}>
+          ${profile.price}/hr
+        </Typography>
+        <Typography className={classes.star}>{getRankInStar(profile.rank)}</Typography>
+        <Grid item>
+          <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
We can move the `LocalizationProvider` into the App.tsx so that it can be reused throughout the app

Reply:
I will delete it, since it is already exist in App.tsx